app/(auth)/sign-up/layout.tsx

```
@@ -0,0 +1,23 @@
+    }
+}
+
+export default function SignupPageLayout({  
+  children,  
+}: Readonly<{  
+  children: React.ReactNode;  
+}>){  
+  return (  
+      <main className="auth-layout auth-layout--login" role="main" aria-label="Login page">  
+      {children}  
+      </main>  
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<main>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)